app/src/main/java/com/neo/speaktouch/utils/NodeValidator.kt

```diff
@@ -120,6 +121,8 @@ object NodeValidator {
 
         if (mustFocus(node)) return false
 
-        return hasContentToRead(node) || hasReadableChild(node)
+        return hasContentToRead(node) ||
+                hasReadableChild(node) ||
+                Type.get(node) == Type.Button
```

Comment:
For now, we will only announce buttons, we can add other scenarios as we identify them.

Reply:
Exactly. As far as I could test, all buttons are being announced properly.